src/homework3/task1.py

```diff
@@ -0,0 +1,16 @@
+""" FizzBuzz
+
+Напишите программу, которая печатает цифры от 1 до 100, но вместо чисел,
+кратных 3 пишет Fizz, вместо чисел кратный 5
+пишет Buzz, а вместо чисел одновременно кратных и 3 и 5 - FizzBuzz
+"""
+
+for digit in range(1, 101):
+    if not digit % 3 and not digit % 5:
```

Comment:
15?

Reply:
print fizzbuzz, i had test it before pushing

Reply:
not digit % 15